applications/neuroscan/frontend/src/components/Sidebar/AddToViewerMenu.jsx

```diff
@@ -33,11 +33,12 @@ const AddToViewerMenu = ({
           const instanceName = anchorEl?.parentNode?.textContent.replace('Add to', '');
           let isEnabled = viewer.config.timePoint === timePoint;
           if (isEnabled) {
-            viewer.config.instances.forEach((instance) => {
-              if (instance.name === instanceName) {
+            const isInstancePresent = (inst) => {
```

Comment:
just added the return to the test function to break the iteration, I just read about findIndex too fast, thx :)

Reply:
@ddelpiano sorry to be a little bit picky ... 
it's not so nice to "misuse" a function to set a variable in another scope.

I mean the `findIndex` is to find the first instance matching the condition. So imo you should manipulate the `isEnabled` outside the scope of the findIndex function. Otherwise it's for other developers difficult to follow the variable `isEnabled` lifespan and changes. In this case the function and the call are closely located. Imagine what will happen if you do this kind of variable manipulation in separate files..... it will not be easy to debug and understand what's going on.

I like self explaining and readable / understandable code